src/embedded.js

```diff
@@ -139,14 +139,15 @@ class Embedded {
 
       await this.ui.init()
       await this.ui.activate()
+      await this.options.load()
 
       console.log('UIController has been activated')
       // Set a body attribute so the content scrip will know if embeded library is active on a page
       this.doc.body.setAttribute('alpheios-embed-lib-status', 'active')
       this.doc.body.addEventListener('Alpheios_Embedded_Check', event => { this.notifyExtension(event) })
       this.doc.body.addEventListener('keydown', event => { this.handleEscapeKey(event) })
-      Object.assign(this.ui.panel.panelData, this.panelData)
-      Object.assign(this.ui.popup.vi.popupData, this.popupData)
+      // Object.assign(this.ui.panel.panelData, this.panelData)
```

Comment:
why commented out?

Reply:
Because these lines completely breaks embed-lib (after Kirill's refactoring components).
This opportunity is not consistent with current modul architecture of components in this way.
I was talking about it on the meeting, so it is a temporary solution to be able to use embed-lib.
Kirill is working on these fixes (as we talked on aTuesday's meeting)

Reply:
yes, you're right. Kirill and I were just talking about them this morning too. I think these will be handled differently going forward.